test_stack.py

```diff
@@ -1,27 +1,33 @@
-import pytest
+import io, pytest, sys
```

Comment:
These should be on separate lines. Also please follow pep 0008 for import ordering.